docs/core_tenets.md

```diff
@@ -0,0 +1,18 @@
+## Jujutsu's Core Tenets
+
+Jujutsu's core tenets are:
+
+ * User-friendliness: Making the  working copy a commit is simpler. This is
+ how the project started.
```

Comment:
nit: indent wrapped lines that belong to the bullet

Reply:
done throughout the document